### DISCUSSION TITLE
Inventree on Proxmox

### DISCUSSION THREAD

**Discussion by rieders** [inventree/InvenTree#5434] [General]
Hello
Many thanks for the help.
I have tried several times to install inventree on Proxmox with more or less success.
I used both ct and vm.
In the two variants I had also tried ubuntu 20.04 and Debian 11.
The installation of Inventree is going well so far and I can access the admin interface.
Now the problem: when I want to install plugins I can't do it in VM.
Even after apt-get update and upgrade Inventree can no longer be reached there. I then only get the Apache default page.

How can I implement a "clean" inventree installation on a Proxmox server and install updates and plugins without any problems?

Does anyone have an Inventree installation running on Proxmox?
What steps do I have to take?
Is there a step by step guide?

Greetings

**Comment by eeintech** [comment]
In my experience Proxmox is just a server management tool, and I only used it to check that my VM running InvenTree was running fine and/or reboot it if needed.
I installed, ran and maintained InvenTree on a Ubuntu VM setup by Proxmox and it was totally fine, you should be able to have everything working like any Ubuntu/Linux machine. The step-by-step documentation provided on the doc site is what you want to follow.

**Comment by rieders** [comment]
Thank you for trying it on a Proxmox server.
Were you able to install plugins without any problems?
The actual installation of Inventree is not the problem.
Adding the plugins is the challenge.

So far I've tried several variations.
Whether it's a CT or VM.
No matter if it is Debian or Ubuntu as Linux.
The problem with the plugin still exists.
I've also noticed that there are differences between debian and Ubuntu at Invenree.
With Ubuntu, a plugin is already installed by the system.
InvenTreeBarcode
InvenTreeCoreNotificationsPlugin
InvenTreeLabel
And a total of 17 plugins are pre-installed

At Debian
InvenTreeBarcode
InvenTreeCoreNotificationsPlugin
And 16 preinstalled plugins.

I hope I can get a system that works well.

Yesterday I created a system again with Ubuntu.
114MB will also be downloaded there. For Debian 109MB.
I then proceeded after this installation.
https://packager.io/gh/inventree/InvenTree/builds/777/install/debian-11
Now Physical Units has also been added under Settings.
Unfortunately, I couldn't install any additional plugins there either.

Greetings
Andre

**Reply by wolflu05** [reply]
Hello, the difference between the two platforms with plugins you noticed are most likely related to different InvenTree versions. The builtin InvenTree label plugin is pretty new.

As @eeintech already said, there should be nothing different setting up InvenTree on proxmox inside a vm (ct I'm not sure, maybe with some permissions, but I don't think so) than setting it up bare metal. Just follow the install guide for bare metal and install inside the vm/ct. 

Like you already said you see the three builtin plugins which means that the first step already looks good. The only think I could imagine is that you didn't [enabled plugin](https://docs.inventree.org/en/latest/start/config/#plugin-options) support. But please provide more info what you mean by you couldn't install additional plugins, otherwise we can't help you.
